aurora-mysql-plugin/pom.xml

```diff
@@ -90,6 +90,35 @@
         <groupId>io.cdap</groupId>
         <artifactId>cdap-maven-plugin</artifactId>
       </plugin>
+      <plugin>
+        <groupId>org.apache.felix</groupId>
+        <artifactId>maven-bundle-plugin</artifactId>
+        <version>5.1.2</version>
+        <extensions>true</extensions>
+        <configuration>
+          <instructions>
+            <_exportcontents>
+              io.cdap.plugin.auroradb.mysql.*;
+              io.cdap.plugin.db.batch.source.*;
+              io.cdap.plugin.db.batch.sink.*;
+              org.apache.commons.lang;
+              org.apache.commons.logging.*;
```

Comment:
What need to export Apache common and Jackson?

Reply:
I simply retained exports from the parent pom. Let me look into this

Reply:
I removed it for mysql, mssql and postgres after verifying but it seems to be coming from the very first PR for each of these plugins - https://github.com/data-integrations/database-plugins/commit/4fb0a7e4469de89bcf25f1cd27a64a31e31c6c09#diff-852a38ca0def089168225bcc77b964c0ec801a99184863eb1b1bc651fa6bc4b0R94

Reply:
I tested 4 plugins, created a jira to test the same for remaining 12 plugins - https://cdap.atlassian.net/browse/PLUGIN-1347